keywording/text.xml

```diff
@@ -337,13 +337,20 @@ So packages are not to be marked stable for one of these architectures.
   <body>
     <p>
       If you maintain an architecture-independent package (data files,
-      icons, pure Perl,...) then you may request that your
+      icons, pure Python, ...) then you may request that your
       package be stabilized on all arches at once. To do this <d/> when
       you are filing the stabilization bug <d/> please add the keyword
       <c>ALLARCHES</c> in addition to <c>STABLEREQ</c> and CC the
       arches that you would like to stabilize.
     </p>
 
+    <p>
+      If your package is <c>ALLARCHES</c> material, you should
```

Comment:
Instead of "ALLARCHES material", I'd prefer the wording "architecture-independent" or "suitable for ALLARCHES stabilization".

Reply:
Sure.